virttest/qemu_devices/qcontainer.py

```diff
@@ -2403,8 +2429,13 @@ def pcic_by_params(self, name, params):
                 bus_length = 20
                 bus_first_port = 1
             elif driver in ('pcie-root-port', 'ioh3420'):
-                bus_length = 1  # multifunction off by default
+                bus_length = 1
                 bus_first_port = 0
+                if driver == 'pcie-root-port':
+                    parent_bus.append({'busid': '_PCI_CHASSIS'})
```

Comment:
Seems that `ioh3420` needs this parent_bus as well.

Reply:
fixed